internal/commands/build/build.go

```diff
@@ -12,6 +12,11 @@ import (
 	"strconv"
 	"strings"
 
+	errors2 "github.com/pkg/errors"
```

Comment:
Just wandering why it's renamed to `errors2`?

Reply:
"errors" package from go runtime and already imported doesn't provide `Wrapf`.